client/src/components/Bills/BillForm.js

```diff
@@ -125,9 +131,9 @@ const BillForm = (props) => {
         />
         %
         <br />
-        Tip Amount: {(bill.subtotal * bill.tip_rate) / 100}
+        Tip Amount: ${GetTipAmount(bill)}
```

Comment:
❗ if it was calculated in the `setState` above, you can just use `tempBill.tip_total` here?

Reply:
GetTipAmount takes the `tip_rate` and the `subtotal`.

If you look at line 28, `tempBill.tip_total = GetTipAmount(tempBill)`

Is that what you mean, or did you want me to do `{GetTipAmount(tempBill)}` on line 134?

Reply:
Changed it to `Tip Amount: ${bill.tip_total}` so it doesn't have to do as much math.